New-language-adaptation/German/training/1-train-acoustic-model.ipynb

```diff
@@ -0,0 +1,553 @@
+    "      # Train dataset  (Concatenate train manifest cleaned and dev manifest cleaned)\n",
+    "      asr_model.cfg.train_ds.manifest_filepath = './data/processed/tar/train/tarred_audio_manifest.json'\n",
+    "      asr_model.cfg.train_ds.is_tarred = True\n",
+    "      asr_model.cfg.train_ds.tarred_audio_filepaths='./data/processed/tar/train/audio_{0..127}.tar'\n",
+    "\n",
+    "      asr_model.cfg.train_ds.batch_size = 32\n",
+    "      asr_model.cfg.train_ds.num_workers = 32\n",
+    "      asr_model.cfg.train_ds.pin_memory = True\n",
+    "      asr_model.cfg.train_ds.trim_silence = True\n",
+    "\n",
+    "      # Validation dataset  (Use test dataset as validation, since we train using train + dev)\n",
+    "      asr_model.cfg.validation_ds.manifest_filepath = ['./data/processed/test_manifest_merged.json', './data/processed/dev_manifest_merged.json']\n",
+    "      asr_model.cfg.validation_ds.batch_size = 32\n",
+    "      asr_model.cfg.validation_ds.num_workers = 32\n",
+    "      asr_model.cfg.validation_ds.pin_memory = True\n",
+    "      asr_model.cfg.validation_ds.trim_silence = True\n",
+    "else:\n",
+    "    # Setup train, validation, test configs\n",
+    "    with open_dict(asr_model.cfg):    \n",
+    "      # Train dataset  (Concatenate train manifest cleaned and dev manifest cleaned)\n",
+    "      asr_model.cfg.train_ds.manifest_filepath = './data/processed/train_manifest_merged.json'\n",
+    "      asr_model.cfg.train_ds.batch_size = 32\n",
+    "      asr_model.cfg.train_ds.num_workers = 32\n",
+    "      asr_model.cfg.train_ds.pin_memory = True\n",
+    "      asr_model.cfg.train_ds.trim_silence = True\n",
+    "\n",
+    "      # Validation dataset  (Use test dataset as validation, since we train using train + dev)\n",
+    "      asr_model.cfg.validation_ds.manifest_filepath = ['./data/processed/test_manifest_merged.json', './data/processed/dev_manifest_merged.json']\n",
+    "      asr_model.cfg.validation_ds.batch_size = 32\n",
+    "      asr_model.cfg.validation_ds.num_workers = 32\n",
+    "      asr_model.cfg.validation_ds.pin_memory = True\n",
+    "      asr_model.cfg.validation_ds.trim_silence = True\n",
+    "\n",
+    "# Point to the new train and validation data for fine-tuning\n",
+    "asr_model.setup_training_data(train_data_config=asr_model.cfg.train_ds)\n",
+    "asr_model.setup_validation_data(val_data_config=asr_model.cfg.validation_ds)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up optimizer and scheduler\n",
+    "\n",
+    "When fine-tuning character models, it is generally advised to use a lower learning rate and reduced warmup. A reduced learning rate helps preserve the pre-trained weights of the encoder. Since the fine-tuning dataset is generally smaller than the original training dataset, the warmup steps would be far too much for the smaller fine-tuning dataset.\n"
```

Comment:
```suggestion
    "When fine-tuning character models, it is generally advised to use a lower learning rate and reduced warmup. A reduced learning rate helps preserve the pretrained weights of the encoder. Since the fine-tuning dataset is generally smaller than the original training dataset, the warmup steps would be far too much for the smaller fine-tuning dataset.\n"
```